### PR TITLE
fix CimCmdlets for #3741

### PR DESCRIPTION
**Pull request by sdwheeler**
This partially addresses the missing content described in #3741.

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [x] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #3741 tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[7b3e770](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/7b3e77066d5331f5e2c6b8e83fd5fd8578db71fa)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[7b3e770](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/7b3e77066d5331f5e2c6b8e83fd5fd8578db71fa)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 48 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[7b3e770](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/7b3e77066d5331f5e2c6b8e83fd5fd8578db71fa)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-5.0&branch=pr-en-us-3742) (powershell-5.0) |
[reference/5.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-5.0&branch=pr-en-us-3742) (powershell-5.0) |
[reference/5.1/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.1/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-5.1&branch=pr-en-us-3742) (powershell-5.1) |
[reference/5.1/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.1/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-5.1&branch=pr-en-us-3742) (powershell-5.1) |
[reference/6/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Export-BinaryMiLog.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Export-BinaryMiLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/export-binarymilog?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Import-BinaryMiLog.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Import-BinaryMiLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/import-binarymilog?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C21%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902210047268601-3742%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=pwbEolbVMFoAIZB0y7N0uUtGIpj8c0DqLKICAmiIIgI%3D&st=2019-02-21T00%3A56%3A11Z&se=2019-03-24T01%3A01%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.0/CimCmdlets/CimCmdlets.md [hunk 1]
```diff
@@ -1,51 +1,65 @@
...
+Download Help Link:
 Locale: en-US
 ---
-
 # CimCmdlets Module
+
 ## Description
-{{Manually Enter Description Here}}
+
+Contains cmdlets that interact with CIM Servers like the WMI service in Windows.
```
Comment (DCtheGeek) [inline reference/5.0/CimCmdlets/CimCmdlets.md]: CIM and WMI should both be spelled out and probably links to more info in first use.

File: reference/5.0/CimCmdlets/CimCmdlets.md [hunk 2]
```diff
@@ -1,51 +1,65 @@
...
 
 ### [New-CimSession](New-CimSession.md)
-{{Manually Enter New-CimSession Description Here}}
+
+Creates a CIM session.
 
 ### [New-CimSessionOption](New-CimSessionOption.md)
-{{Manually Enter New-CimSessionOption Description Here}}
+
+Specifies advanced options for the New-CimSession cmdlet.
```
Comment (DCtheGeek) [inline reference/5.0/CimCmdlets/CimCmdlets.md]: cmdlet should be code fenced.

File: reference/5.0/CimCmdlets/CimCmdlets.md [hunk 3]
```diff
@@ -1,51 +1,65 @@
...
 
 ### [Remove-CimSession](Remove-CimSession.md)
-{{Manually Enter Remove-CimSession Description Here}}
+
+Removes one or more CIM sessions.
 
 ### [Set-CimInstance](Set-CimInstance.md)
-{{Manually Enter Set-CimInstance Description Here}}
+
+Modifies a CIM instance on a CIM server by calling the ModifyInstance method of the CIM class.
```
Comment (DCtheGeek) [inline reference/5.0/CimCmdlets/CimCmdlets.md]: ModifyInstance should probably be bolded as a method.

File: reference/5.0/CimCmdlets/Register-CimIndicationEvent.md [hunk 4]
```diff
@@ -13,120 +14,132 @@ Subscribes to indications using a filter expression or a query expression.
...
-@{navigationLink=System.Management.Automation.PSObject[]; #text=System.Management.Automation.PSObject[]}
-```
-
-This set of commands subscribes to the events generated by the class named Win32_ProcessStartTrace which raises an event whenever a process starts, and gets the events with ProcessStarted subscription using the Get-Event cmdlet.
-For more information about Get-Event, see http://technet.microsoft.com/library/hh849909.aspx.
 
-Note: for this example, you must run PowerShell as an Administrator.
+This set of commands subscribes to the events generated by the class named **Win32_ProcessStartTrace**
+which raises an event whenever a process starts, and gets the events with ProcessStarted
+subscription using the `Get-Event` cmdlet. For more information see [Get-Event](../Microsoft.PowerShell.Utility/Get-Event.md).
```
Comment (DCtheGeek) [inline reference/5.0/CimCmdlets/Register-CimIndicationEvent.md]: Comma after 'For more information'

File: reference/5.0/CimCmdlets/Register-CimIndicationEvent.md [hunk 5]
```diff
@@ -265,67 +289,65 @@ Accept wildcard characters: False
...
-Specifies a query to run on the CIM server.
-
-You can specify the query dialect using the QueryDialect parameter.
 
-If the value specified contains double quotes ("), single quotes ('), or a backslash (\\), you must escape those characters by prefixing them with the backslash (\\) character.
-If the value specified uses the WQL LIKE operator, then you must escape the following characters by enclosing them in square brackets (\[\]): percent (%), underscore (_), or opening square bracket (\[).
+Specifies a query to run on the CIM server. If the value specified contains double quotes ("),
+single quotes ('), or a backslash (\\), you must escape those characters by prefixing them with the
+backslash (\\) character. If the value specified uses the WQL LIKE operator, then you must escape
+the following characters by enclosing them in square brackets (\[\]): percent (%), underscore (_),
```
Comment (DCtheGeek) [inline reference/5.0/CimCmdlets/Register-CimIndicationEvent.md]: I think you need to markdown escape the underscore or remaining content is in italics.

**Comment by DCtheGeek** [comment]
Looks like the build failure was due to the markdown issue mentioned.  Fixing that should fix the build.

```
New-ExternalHelpCab : Cannot bind argument to parameter 'URI' because it is an empty string.
At D:\a\1\s\build.ps1:117 char:28
+ ...  $cabInfo = New-ExternalHelpCab -CabFilesFolder $MamlOutputFolder -La ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [New-ExternalHelpCab], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,New-ExternalHelpCab
 
New-ExternalHelpCab : Cannot bind argument to parameter 'URI' because it is an empty string.
At D:\a\1\s\build.ps1:117 char:28
+ ...  $cabInfo = New-ExternalHelpCab -CabFilesFolder $MamlOutputFolder -La ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [New-ExternalHelpCab], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,New-ExternalHelpCab
 
There are errors during platyPS run!
Please fix your markdown to comply with the schema: https://github.com/PowerShell/platyPS/blob/master/platyPS.schema.md
At D:\a\1\s\build.ps1:133 char:5
+     throw "There are errors during platyPS run!`nPlease fix your mark ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (There are error...atyPS.schema.md:String) [], RuntimeException
    + FullyQualifiedErrorId : There are errors during platyPS run!
    Please fix your markdown to comply with the schema: https://github.com/PowerShell/platyPS/blob/master/platyPS.sche 
   ma.md
```

File: reference/5.0/CimCmdlets/CimCmdlets.md (hunk 1 above)
Comment (sdwheeler) [reply reference/5.0/CimCmdlets/CimCmdlets.md]: Expanded the TLAs but didn't link. Assume that they know why they are here.

File: reference/5.0/CimCmdlets/CimCmdlets.md (hunk 2 above)
Comment (sdwheeler) [reply reference/5.0/CimCmdlets/CimCmdlets.md]: fixed

File: reference/5.0/CimCmdlets/CimCmdlets.md (hunk 3 above)
Comment (sdwheeler) [reply reference/5.0/CimCmdlets/CimCmdlets.md]: fixed

File: reference/5.0/CimCmdlets/Register-CimIndicationEvent.md (hunk 4 above)
Comment (sdwheeler) [reply reference/5.0/CimCmdlets/Register-CimIndicationEvent.md]: done

File: reference/5.0/CimCmdlets/Register-CimIndicationEvent.md (hunk 5 above)
Comment (sdwheeler) [reply reference/5.0/CimCmdlets/Register-CimIndicationEvent.md]: fixed

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9ffd251](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/9ffd251ba97d909911a2b458487cea4ed24e40d7)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9ffd251](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/9ffd251ba97d909911a2b458487cea4ed24e40d7)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 48 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9ffd251](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/9ffd251ba97d909911a2b458487cea4ed24e40d7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-5.0&branch=pr-en-us-3742) (powershell-5.0) |
[reference/5.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-5.0&branch=pr-en-us-3742) (powershell-5.0) |
[reference/5.1/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.1/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-5.1&branch=pr-en-us-3742) (powershell-5.1) |
[reference/5.1/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.1/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-5.1&branch=pr-en-us-3742) (powershell-5.1) |
[reference/6/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Export-BinaryMiLog.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Export-BinaryMiLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/export-binarymilog?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Import-BinaryMiLog.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Import-BinaryMiLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/import-binarymilog?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C21%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902211502333749-3742%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=w1nLJ9iBigHnT1WrG4v2uKkwYQ1DZv9ojhUqhPdtOdI%3D&st=2019-02-21T15%3A11%3A12Z&se=2019-03-24T15%3A16%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by DCtheGeek** [comment]
@sdwheeler All changes look fine, but looks like the build validation is still failing.  Looks like an issue with a non-included file, however:

```
    Directory: D:\a\1\s\maml\6\PSReadLine


Mode                LastWriteTime         Length Name                                                                  
----                -------------         ------ ----                                                                  
-a----        2/21/2019   3:24 PM         105652 Microsoft.PowerShell.PSReadLine.dll-Help.xml                          
D:\a\1\s\updatablehelp\6\PSReadLine\PSReadLine_5714753b-2afd-4492-a5fd-01d9e2cff8b5_HelpInfo.xml
```

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[33e8e8a](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/33e8e8ad3693118e13dd0340243e9dfe789d715a)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[33e8e8a](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/33e8e8ad3693118e13dd0340243e9dfe789d715a)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 49 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[33e8e8a](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/33e8e8ad3693118e13dd0340243e9dfe789d715a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-5.0&branch=pr-en-us-3742) (powershell-5.0) |
[reference/5.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-5.0&branch=pr-en-us-3742) (powershell-5.0) |
[reference/5.1/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.1/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-5.1&branch=pr-en-us-3742) (powershell-5.1) |
[reference/5.1/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.1/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-5.1&branch=pr-en-us-3742) (powershell-5.1) |
[reference/6/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Export-BinaryMiLog.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Export-BinaryMiLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/export-binarymilog?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Import-BinaryMiLog.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Import-BinaryMiLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/import-binarymilog?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/PSReadLine/PSReadLine.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/PSReadLine/PSReadLine.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psreadline/index?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C21%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902211933336299-3742%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=v9dnpMTU55lNxuCUxArgybIVJgl%2BVN0tCOU%2FwPMpHhU%3D&st=2019-02-21T19%3A42%3A15Z&se=2019-03-24T19%3A47%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a5f0d78](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/a5f0d78ced748c087d4c091b2cfefb6a04bd4d2a)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a5f0d78](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/a5f0d78ced748c087d4c091b2cfefb6a04bd4d2a)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 49 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a5f0d78](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/a5f0d78ced748c087d4c091b2cfefb6a04bd4d2a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-5.0&branch=pr-en-us-3742) (powershell-5.0) |
[reference/5.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-5.0&branch=pr-en-us-3742) (powershell-5.0) |
[reference/5.1/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.1/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-5.1&branch=pr-en-us-3742) (powershell-5.1) |
[reference/5.1/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.1/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-5.1&branch=pr-en-us-3742) (powershell-5.1) |
[reference/6/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Export-BinaryMiLog.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Export-BinaryMiLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/export-binarymilog?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Import-BinaryMiLog.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Import-BinaryMiLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/import-binarymilog?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/PSReadLine/PSReadLine.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/PSReadLine/PSReadLine.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psreadline/index?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C21%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902211933394434-3742%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JhzCoUiQfwziwoExKBcOu1xvlMfySsERr%2BQYoQqboC0%3D&st=2019-02-21T19%3A55%3A26Z&se=2019-03-24T20%3A00%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[15d1096](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/15d10962b7011a9d6326d018435629a9a7e8c9ba)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[15d1096](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/15d10962b7011a9d6326d018435629a9a7e8c9ba)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 49 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[15d1096](https://github.com/sdwheeler/PowerShell-Docs/pull/3742/commits/15d10962b7011a9d6326d018435629a9a7e8c9ba)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-5.0&branch=pr-en-us-3742) (powershell-5.0) |
[reference/5.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-5.0&branch=pr-en-us-3742) (powershell-5.0) |
[reference/5.1/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.1/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-5.1&branch=pr-en-us-3742) (powershell-5.1) |
[reference/5.1/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/5.1/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-5.1&branch=pr-en-us-3742) (powershell-5.1) |
[reference/6/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Export-BinaryMiLog.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Export-BinaryMiLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/export-binarymilog?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Import-BinaryMiLog.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Import-BinaryMiLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/import-binarymilog?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |
[reference/6/PSReadLine/PSReadLine.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3741-p1/reference/6/PSReadLine/PSReadLine.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psreadline/index?view=powershell-6&branch=pr-en-us-3742) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C21%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902212025566283-3742%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hgHVrV1KxEu0GUAvXrbfvMf7qQzJo98muYZ2GQbT4Ls%3D&st=2019-02-21T20%3A33%3A57Z&se=2019-03-24T20%3A38%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sdwheeler** [comment]
Abandoning this PR. Files fixed in another PR.

**Pull request closed without merging**